src/lib/scheduler.js

```diff
@@ -119,10 +119,11 @@ const getDefaultSemesterIndex = () => {
   let result = 0;
 
   const now = new Date();
+  console.log(now);
   // Check if Winter (Period 3, above)
   if (
     new Date(DATES.PREREG.WINTER) < now &&
-    now < addDays(new Date(DATES.Winter.END), -7)
+    now < addDays(new Date(DATES.Winter.START))
```

Comment:
Hmmm... I think this is a bit too aggressive.
Maybe change it to 7 days after Winter Study starts (spring registration usually starts the second week of WS).

Also, don't forget to updates the comments on L114 to reflect this logic change.